src/utils/rss-generation.js

```diff
@@ -0,0 +1,65 @@
+import sanitizeHtml from 'sanitize-html'
+import removeMd from 'remove-markdown'
+
+const generateXMLTag = (tag, attrs = {}, content = '') => {
+  const attrsStr = Object.entries(attrs)
+    .map(([key, value]) => `${key}="${value}"`)
+    .join(' ').trim()
+  return `<${tag}${attrsStr ? ` ${attrsStr}` : ''}>${content}</${tag}>`
+}
+
+const generateXMLTags = (tags) => {
+  return tags.map(tag => generateXMLTag(tag.name, tag.attrs, tag.content)).join('\n')
+}
```

Comment:
This is never used, so should it be removed?

Reply:
I think we can leave it as is, in case it being useful in the future.